frontend/public/components/modals/column-management-modal.tsx

```diff
@@ -26,35 +26,40 @@ const readOnlyColumns = new Set([NAME_COLUMN_ID]);
 const DataListRow: React.FC<DataListRowProps> = ({
   checkedColumns,
   column,
-  onChange,
   disableUncheckedRow,
-}) => (
-  <DataListItem
-    aria-labelledby={`table-column-management-item-${column.id}`}
-    key={column.id}
-    className="pf-c-data-list__item--transparent-bg"
-  >
-    <DataListItemRow>
-      <DataListCheck
-        isDisabled={
-          (disableUncheckedRow && !checkedColumns.has(column.id)) || readOnlyColumns.has(column.id)
-        }
-        aria-labelledby={`table-column-management-item-${column.id}`}
-        checked={checkedColumns.has(column.id)}
-        name={column.title}
-        id={column.id}
-        onChange={onChange}
```

Comment:
Does removing `onChange` break selection with a keyboard or screenreader?

Reply:
@spadgett simplified solution, and kept onChange